bin/lando/setup.sh

```diff
@@ -55,7 +55,7 @@ echo_heading "Installing WordPress $WP_VERSION"
 wp core install \
 	--path=$LANDO_WEBROOT \
 	--url="http://vip-go-dev.lndo.site" \
-	'--title="VIP Go Dev"' \
+	--title="VIP Go Dev" \
 	--admin_user="vipgo" \
```

Comment:
I've removed this quotes as they were creating the WordPress site with the title "VIP Go Dev", quotes included